src/rules/enum_values_all_caps.js

```diff
@@ -0,0 +1,18 @@
+import { GraphQLError } from 'graphql/error';
+
+export function EnumValuesAllCaps(context) {
+  return {
+    EnumValueDefinition(node, key, parent, path, ancestors) {
+      const fieldName = node.name.value;
```

Comment:
Sure, I'd just copied from here: https://github.com/cjoudrey/graphql-schema-linter/blob/master/src/rules/deprecations_have_a_reason.js#L39.  Will change in both spots.